terraform/backend/main.tf

```diff
@@ -21,11 +21,11 @@ resource "aws_s3_bucket_logging" "terraform_state" {
   bucket = aws_s3_bucket.terraform_state.id
 
   target_bucket = aws_s3_bucket.state_access_log_bucket.id
-  target_prefix = "production/"
+  target_prefix = "staging/"
 }
 
 resource "aws_s3_bucket" "terraform_state" {
-  bucket = "data-collection-service-tfstate-dev"
+  bucket = "data-collection-service-tfstate-sandbox"
```

Comment:
I'm not sure what this change is about

Reply:
Ah sorry, separate to this ticket, Seun asked me to try creating stuff in his new sandbox environment to see if it works. Said he'd follow up with you

Reply:
I haven't heard anything, shall I just leave it on this branch for now?